bentoml/service.py

```diff
@@ -299,6 +299,24 @@ def decorator(func):
     return decorator
 
 
+def webui_decorator(web_static_content):
+    """Define web UI static files required to be bundled with a BentoService
+
+    Args:
+        web_static_content: path to directory containg index.html and static dir
+
+    >>>  @webui('./ui/')
```

Comment:
I'm a bit inclined to just name it `@web_static_content('./ui/')` as the user-facing API, thoughts?

Reply:
I am fine either way, will change it to `web_static_content`

Reply:
sounds good! I was thinking that users can potentially use this to host other files other than web UI files, e.g., using this to host test input files, JSON config file(some use cases require exposing server-side config to the client), or even pdf documentation, etc.

Reply:
Makes sense, the only downside is users currently cannot specify custom `/` path. So by default `bento` will look for index.html.
So for example, if someone is trying to serve `index.json` at `/`, then this won't work but `/index.json` will work.

We can add option to customize this, but I am more inclined not to as for production deployment it is better to just use `nginx` or something as serving static files via flask is both slow and processor intensive.